Packs/GitHub/Integrations/GitHub/GitHub.yml

```
@@ -2462,7 +2613,7 @@ script:
     - contextPath: GitHub.ActionsUsage.WorkflowUsage
       description: GitHub action worflow usage on different OS.
       type: Unknown
-  dockerimage: demisto/pyjwt3:1.0.0.10828
+  dockerimage: demisto/pyjwt3:1.0.0.15504
```

```suggestion
  dockerimage: demisto/pyjwt3:1.0.0.10828
```

please o not bump version, it will break the integration. I will force merge this validation failure.
 see https://github.com/demisto/etc/issues/33818